example/monitoring-stats.txt

```diff
@@ -6,10 +6,6 @@ io.cmd
 io.cmd.{<any elliptics command type in upper case>}
     time of commands for each type
 
-io.cmd.{<any elliptics command type in upper case>}.lock_time
-    time spent in id lock waiting, for each command type,
-    (this will be 0 for commands with DNET_FLAGS_NOLOCK flag set)
-
```

Comment:
If there are only already locked keys in the queue, then i/o pool threads will wait for conditional variable (dnet_request_queue::m_queue_wait), but they will not wait for any lock.